### PR TITLE
responsiveness

### PR DESCRIPTION
**Pull request by preethasri**
add responsiveness for mobile and laptop

### CODE REVIEW

File: src/components/categories/Categories.css [hunk 1]
```diff
@@ -5,32 +5,34 @@
...
+    
 
 }
 .categoryItem-container{
       flex:1;
       margin: 10px;
       height:50vh;
       position: relative;
-      width:300px;
+      width:400px;
```
Comment (bipin7yadav) [inline src/components/categories/Categories.css:18]: you can use rem instead of pixel

**Review by bipin7yadav — APPROVED** [review]
advice : you should start using rem instead of px .

File: src/components/categories/Categories.css [hunk 2]
```diff
@@ -5,32 +5,34 @@
...
+      
     
 }
 .category-img{
-    height:50vh;
+    height:30vh;
     margin:3px;
    position: relative;
-    width:300px;
+    width:400px;
```
Comment (KaustubhM1997) [inline src/components/categories/Categories.css:28]: Suggestion: For anything more than 10px, the calculations should be in rem

File: src/components/navbar/navbar.css [hunk 3]
```diff
@@ -126,12 +126,27 @@
...
-    width: 50px;
-    height: 50px;
-    font-size: 12px;
+    
+    width: 25px;
+    height: 20px;
+    border-radius: 50%;
+   display: block;
+    position: absolute;
+    right: -18px;
```
Comment (KaustubhM1997) [inline src/components/navbar/navbar.css:135]: The shorthand property can be used here.

File: src/pages/cartmanagement/cartmanagement.css [hunk 4]
```diff
@@ -30,15 +29,17 @@
...
   
   .imganddetails{
     display:flex;
-    height:220px;
-    max-width:100%;
-    margin: auto;
-    border:1px solid black;
+  width:800px;
+  height:280px;
+  margin-left:8rem;
```
Comment (KaustubhM1997) [inline src/pages/cartmanagement/cartmanagement.css:39]: The shorthand property for margin can be used here.

**Review by KaustubhM1997 — APPROVED** [review]
Looks good to merge!

File: src/components/categories/Categories.css (hunk 1 above)
Comment (preethasri) [reply src/components/categories/Categories.css:18]: will work on this👍

**Pull request merged**